### ISSUE TITLE
Unable to remove text using plot_channel_maps

### ISSUE THREAD

**Issue by drtobybrown** [radio-astro-tools/spectral-cube#689]
From the [docs](https://spectral-cube.readthedocs.io/en/latest/api/spectral_cube.SpectralCube.html#spectral_cube.SpectralCube):
> plot_channel_maps(...)
>      **textcolor**: None or str
>          _Color of the label text to overlay. If None, will be determined automatically. If 'notext', no text will be added._

However in the following example this gives the error `ValueError: Invalid RGBA argument: 'notext'`

Example:

```
from astropy.utils import data
from spectral_cube import SpectralCube

fn = data.get_pkg_data_filename('tests/data/example_cube.fits', 'spectral_cube')
cube = SpectralCube.read(fn)

cube.plot_channel_maps(2, 2, np.arange(1,5), textcolor='notext')
```

spectral-cube version 0.5.0

**Comment by keflavich** [comment]
looks like this line:
https://github.com/radio-astro-tools/spectral-cube/blob/71fd8e2c7251b46e2738af6fa3c0b78fd242260c/spectral_cube/spectral_cube.py#L3421
should have been in the `if textcolor != 'notext'` indented block

**Comment by drtobybrown** [comment]
Yep, I see. 

I also notice that `contourcolors` kwarg is no longer in the function (incorporated into `contourkwargs={}`?) but remains in the docs and comments.

**Comment by keflavich** [comment]
We'd welcome a PR to fix these!

**Comment by drtobybrown** [comment]
done.

**Comment by keflavich** [comment]
closed by #690
